packages/contentstack-clone/src/lib/util/clone-handler.js

```diff
@@ -1,12 +1,12 @@
 const ora = require('ora');
 const path = require('path');
 const inquirer = require('inquirer');
-const rimraf = require('rimraf');
 const chalk = require('chalk');
+const prompt = require('prompt');
+const colors = require('@colors/colors/safe');
```

Comment:
@vkalta I saw, you have used colors package items in the code but I don't see the colors package in package.json, can you add that otherwise this can be failed while installing and releasing the individual package.